rtf_vis_tool/src/Components/frustum_classes/se3.js

```diff
@@ -51,6 +51,22 @@ class SE3 {
                             point_world_coords.get(0,2)]])
         return point_world_coords;
     }
+
+    inverse() {
+        /* Computes the inverse of the current pose matrix. (Since numjs doesn't have a function for it)
+
+        Returns:
+            A new SE3 object storing the inverse of the current pose matrix.
+        */
+        const Rt = this.rotation.T;
+        const t_negative = nj.array([-1 * this.translation.get(0), 
+                              -1 * this.translation.get(1), 
+                              -1 * this.translation.get(2)]);
+
+        const t_inverse = Rt.dot(t_negative);
+        const inverse = new SE3(Rt, t_inverse);
+        return inverse;
```

Comment:
I think this would be cleaner just as
```js
return new SE3(Rt, Rt.dot(t_negative))
```

Reply:
or if we want the const, which we probably do, i would prefer:
```js
const inverse = new SE3(Rt, Rt.dot(t_negative));
return inverse;
```

Reply:
yup that looks cleaner